### DISCUSSION TITLE
Cool Feature's

### DISCUSSION THREAD

**Discussion by Mofex1** [modrinth/.github#289] [Feature requests]
please add these feature to make modrinth more cool 

1. a**dd change log or updates log in settings tab to see the recently update to see what changed (added or removed) for nerd people like me 
and pop-up window when new update installed like this:**
![image](https://github.com/user-attachments/assets/42056ac5-c7dd-4ff4-8326-2f1fbc9ed323)

2. **make the update all button update the projects into matching version of profile**

3. **make button that's called "Copy" that's copy the selected project when in select mode to paste it on another profile without copy whole profile**

4. **make ability to run multiple profiles in same time who have more than 1 account (if this possible)** 

5. **make a filter button on profiles that's can filter into "resource packs, mods, shades, data pack"**

**Comment by Thinkseal** [comment]
Sadly optifine can't be added because the creator of optifine wants there website to be the only one that hosts the mod

**Reply by Mofex1** [reply]
> Sadly optifine can't be added because the creator of optifine wants there website to be the only one that hosts the mod



> Sadly optifine can't be added because the creator of optifine wants there website to be the only one that hosts the mod

that's really sad

**Comment by Thinkseal** [comment]
I believe you can already do number five unless you're referring to booting one profile multiple times (without closing the game) in which case that could corrupt some files so it will not be added
